docs/theory/diffusion/index.rst

```diff
@@ -0,0 +1,25 @@
+Diffusion
+=========
+
+Pylake supports a number of methods for studying diffusive processes.
+These methods rely on quantifying the movement of the fluorophore.
```

Comment:
Without context (explaining about a kymograph, optical tweezers), this sentence can be a bit confusing. Currently, this context is given in the introduction to diffusion analysis. You could just remove the sentence from this paragraph.